src/d3/scatterplot.d3.ts

```diff
@@ -10,46 +10,110 @@ export interface ScatterPlotStyle {
 }
 
 export class Scatterplot{
+    readonly AILERON_FONT = 'https://cdn.aframe.io/fonts/Aileron-Semibold.fnt';
+    readonly DATA_PT_RADIUS = .05;
+    private GRID_BOUND = 50;
     private data: VRScatterPoint[];
     private shape: string;
     private container: HTMLElement | null;
-    private TIME_MAX = 31;
-    timeScale: d3.ScaleTime<number, number>;
+    private dataPointContainer: HTMLElement;
+    private dataTextContainer: HTMLElement;
+    xScale: d3.ScaleLinear<number, number>;
+    yScale: d3.ScaleLinear<number, number>;
+    zScale: d3.ScaleLinear<number, number>;
     dataType: MetaType;
 
+
   constructor(shape: string) {
     this.shape = shape;
   }
   init(container: HTMLElement, data: VRScatterPoint[], dataType: MetaType){
-    // console.log(data[0].categories.browser);
     this.data = data;
+    console.log(data);
     this.dataType = dataType;
     this.container = container;
     this.dataType = dataType;
-    this.generatePts();
-    this.setColor('blue');
+    this.dataPointContainer = document.createElement('a-entity');
```

Comment:
Can probably abstract this pattern a bit to a helper since it's repeated twice.

Reply:
Made a function called `createEntity` that takes in a `selector` and `className` and then returns an `HTMLElement`